config/portals.yml

```diff
@@ -64,6 +64,19 @@ portals: &portals
     scripts:
       - edsc-portal.cwic.js
 
+  IDN:
```

Comment:
To match the rest of the file please make this lower case

Reply:
You want me to add the following (?):
    scripts:
      - edsc-portal.idn.js

Reply:
IDN should be idn, search 3 has the same policy.

Reply:
@mmorahan this still needs to be cleaned up. If you have made the change locally, please push that commit. I'll move this back to "unresolved" for the moment since the change isn't on Github yet.